index.html

```diff
@@ -1205,12 +1217,12 @@ <h2>Bindings for Events</h2>
       </section>
 
     </section>
-
+<!-- 
     <section id="processing-model" class="informative">
       <h1>Processing Model and Runtime</h1>
       <p>
         This section describes the processing model for Protocol Bindings with
-        respect to the abstract WoT Interactions provided by the Scripting API.
+        respect to the abstract WoT operations provided by the Scripting API.
```

Comment:
The Scripting API is just an API, it does not _provide_ any abstract operations, eventually it _describes_ the API provided by the runtime (WoT Servient).

Reply:
You are right, but this section is removed in the end.